src/dev/build/tasks/clean_tasks.js

```diff
@@ -255,3 +256,38 @@ export const CleanEmptyFoldersTask = {
     );
   },
 };
+
+export const CleanOtherPlatformFilesFromModulesTask = {
+  description: 'Cleaning files of other platforms or unnecessary sub-packages from node_modules',
+
+  async run(config, log, build) {
+    const getExtraFilesInNuclide = platform => {
+      const packageRoot = build.resolvePath('node_modules/nuclide-prebuilt-libs');
+      // Delete unnecessary packages in nuclide-prebuilt-libs
+      const paths = [];
+      paths.push(resolve(packageRoot, 'fuzzy-native'));
+      paths.push(resolve(packageRoot, 'keytar'));
+      paths.push(resolve(packageRoot, 'pty'));
+      paths.push(resolve(packageRoot, '**/ctags-electron-*'));
+      // Delete extra files from platform-specific builds
+      if (platform.isWindows()) {
+        paths.push(resolve(packageRoot, '**/ctags-*-darwin-x64'));
```

Comment:
![image](https://user-images.githubusercontent.com/1646270/60294370-0e309900-98d6-11e9-9395-a6fab3dd4753.png)

we don't support all node version, so we can delete some more

I suggest we construct the filename that needed in this release and delete others 
I don't n

Reply:
you could do something similar to this https://github.com/facebook-atom/nuclide-prebuilt-libs/blob/master/find-binary.js#L47

you can actually probably direct use this API

Reply:
But they don't build these binaries in our node version(i.e. 10.15.2), but these binaries are compatible with our version, so which version should we keep?

Reply:
https://github.com/facebook-atom/nuclide-prebuilt-libs/blob/master/find-binary.js#L32 nodeAbi iscomputed according to current node process, so I think use it is, I think our abi version is 57, but since those are subject to change let's use api to decide

Reply:
Done.